reference/metric.py

```diff
@@ -355,9 +355,9 @@ def score(pred, gt, shore_root, distance_tolerance=200, shore_tolerance=2):
                 scene_id,
                 shore_tolerance + distance_tolerance / 1000,
             )
-            print(
-                f"{len(gt_sc_shore)} ground truth, {len(pred_sc_shore)} predictions close to shore"
-            )
+            #print(
+            #    f"{len(gt_sc_shore)} ground truth, {len(pred_sc_shore)} predictions close to shore"
+            #)
```

Comment:
Any particular reason to remove this?

Reply:
The constant unbuffered prints to stdout really slow down the scoring. It also messes with the `tqdm` formatting.

Reply:
Approving this after manual testing with a set of dummy files.  Repro'd issue, confirm that fix works.

Reply:
Makes sense, agree, let's leave it commented in case anyone wants to use it.